tools/style.sh

```diff
@@ -3,20 +3,28 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-set -o errexit
+set +o errexit
```

Comment:
May I understand why it turns off?

Reply:
I unset it so that it will always run all checks in the script. The `$err` will set the exit code if any of them fails